cypress/support/bodyTestHelper.js

```diff
@@ -187,3 +187,18 @@ export const errorTitle = service => {
     `${service.translations.error[404].title} - ${service.brandName}`,
   );
 };
+
+export const hasNoscriptImgAtiUrl = () => {
+  getElement('noscript')
+    .eq(0)
+    .should(
+      'contain',
+      '<img height="1px" width="1px" alt="" src="https://a1.api.bbc.co.uk/hit.xiti?',
```

Comment:
Can we have a test that asserts, on the different environments, the beacon includes the correct bucket ID?

Reply:
We could - that'd work well for the world service sites which go into the same bucket regardless of whether they're being accessed in the UK or outside of the UK.

For News - when outside of the UK, (on Travis CI, Jenkins CI and if any developer using this application is outside of the UK) the buckets will be `*GNL*`, but in the UK the buckets will be `*PS*`. Given the likelihood of getting false negatives, I didn't add this Cypress test to our test suite. 

| Service | Env | isUK | Bucket |
| -- |-- |--|--|
|`news` | `test`| `true` | `NEWS_PS_TEST`|
|`news` | `test`| `false` | `NEWS_GNL_TEST`|
|`news` | `live`| `true` | `NEWS_PS`|
|`news` | `live`| `false` | `NEWS_GNL`|
|`persian` <br /> (and any other World Service site) | `test`| `true` or `false` | `WS_NEWS_LANGUAGES_TEST`|
|`persian` <br />  (and any other World Service site) | `live`| `false` or `false` | `WS_NEWS_LANGUAGES`|

Reply:
I've now added the Cypress tests for the World service buckets - [commit is here](https://github.com/bbc/simorgh/pull/1651/commits/e45e4c6f69b2b5e92fc7c8c218244a034fe333e0). I haven't added the News buckets for the reasons explained above.